packages/web/src/services/Location.ts

```diff
@@ -23,8 +23,14 @@ export type Location = {
 
 export const getLocation = async (): Promise<Location | null> => {
   try {
+    const cached_location = localStorage.getItem('ipapi_location')
```

Comment:
Would an in memory thing here be better? I potentially worry about persisting this across sessions and forgetting about it or something.

NBD though, this seems fine as is.

Reply:
Yeah I agree @raymondjacobson, we only need this per session so let's just stick it in memory

Reply:
yea had the same thought after publishing this but was passing out earlier, updated now!